docs/pages/2019_MARVEL_Psik_MaX/sections/verdi_shell.rst

```diff
@@ -105,55 +105,55 @@ show what elements it corresponds to by typing:
 
 .. code:: python
 
-     upf = load_node(PK)
-     upf.element
+    upf = load_node(PK)
+    upf.element
 
 All methods of ``UpfData`` are accessible by typing ``upf.`` and then pressing ``TAB``.
 
 k-points
 ~~~~~~~~
 
 A set of k-points in the Brillouin zone is represented by an instance of the
-``KpointsData`` class. Choose one from the graph of 
+``KpointsData`` class. Choose one from the graph of
 produced in section :ref:`aiidagraph`,
 load it as ``kpoints`` and inspect its content:
 
 .. code:: python
 
-     kpoints.get_kpoints_mesh()
+    kpoints.get_kpoints_mesh()
 
 Then get the full (explicit) list of k-points belonging to this mesh using
 
 .. code:: python
 
-     kpoints.get_kpoints_mesh(print_list=True)
+    kpoints.get_kpoints_mesh(print_list=True)
 
-If you incurred in a ``AttributeError``, it means that the kpoints instance
-does not represent a regular mesh but rather a list of k-points defined by
-their crystal coordinates (typically used when plotting a band structure).
+If this throws an ``AttributeError``, it means that the kpoints instance does not represent a regular mesh but rather a list of k-points defined by their crystal coordinates (typically used when plotting a band structure).
```

Comment:
is there a simple way for them to figure out what type of k-points it is *before* they get an error?

Reply:
Sure, but this will almost definitely make it more complex than we want it to be and so will be shifting the problem I feel

Reply:
ok.